zerver/lib/email_mirror.py

```diff
@@ -161,12 +162,23 @@ class ZulipEmailForwardUserError(ZulipEmailForwardError):
     pass
 
 def send_zulip(sender: str, stream: Stream, topic: str, content: str) -> None:
+    trunc_topic = truncate_topic(topic)
+    if len(trunc_topic) < len(topic):
+        pdlte = datetime.datetime.today()
+        pdgt = pdlte - datetime.timedelta(days = 7)
+        if not (Message.objects.filter(
+                subject = trunc_topic,
+                recipient = get_stream_recipient(stream.id),
+                date_sent__lte = pdlte,
+                date_sent__gt = pdgt).exists()):
+            content = 'Subject: ' + topic + '\n\n' + content
```

Comment:
Using string formatting instead of arithmetic should be more readable:
``'Subject: {}\n\n'.format(topic, content)``